starter-code/src/functions-and-arrays.js

```diff
@@ -1,4 +1,29 @@
 // Find the maximum
+/**
+ * Returns the largest of two numbers
+ * @param x first number to compare
+ * @param y second number to compare
+ */
+function maxOfTwoNumbers(x, y){
+
+  return (x < y)? y: x;
```

Comment:
You're not supposed to know that (? : ;) at that stage of the bootcamp but that's perfect :) well done!